### PR TITLE
Improve wording on Development version header

### PR DESCRIPTION
**Pull request by kkaempf**
Signed-off-by: Klaus Kämpf <kkaempf@suse.de>

### CODE REVIEW

**Comment by rdoxenham** [comment]
Great, thanks!

**Review by Itxaka — APPROVED** [review]
should we add a note that stable only supports x86?

**Review by davidcassany — APPROVED** [review]

File: docs/quickstart.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ helm upgrade --create-namespace -n cattle-elemental-system --install elemental-o
 ```
 
 </TabItem>
-<TabItem value="develOperator" label="Development version (for ARM)" default>
+<TabItem value="develOperator" label="Development version (including ARM)" default>
```
Comment (Itxaka) [inline docs/quickstart.md:81]:
```suggestion
<TabItem value="develOperator" label="Development version (x86, ARM(64?))" default>
```

Comment (Itxaka) [reply docs/quickstart.md:81]: and on line 74:

```
<TabItem value="stableOperator" label="Stable version (x86)" default>
```

**Comment by kkaempf** [comment]
Thanks for the feedback - is the current version better ?

**Review by Itxaka — APPROVED** [review]

**Pull request merged**